templates/patterns/banners/pattern-banner.html.twig

```diff
@@ -70,6 +85,11 @@
     full_width: full_width,
     credit: credit|default(''),
     extra_classes: extra_classes|default(''),
+    credit: credit|default(''),
+    video: _video,
+    sr_play: sr_play,
+    sr_pause: sr_pause,
```

Comment:
We should always have values for screen readers, so if they're not passed to the pattern let's set them here.